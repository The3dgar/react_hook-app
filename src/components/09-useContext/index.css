body {
  padding: 0px;
  background-color: white;
  color: black
}