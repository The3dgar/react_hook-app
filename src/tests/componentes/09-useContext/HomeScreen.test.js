import React from "react";
import { shallow, mount } from "enzyme";
import { UserContext } from "../../../components/09-useContext/UserContext";
const { HomeScreen } = require("../../../components/09-useContext/HomeScreen");

describe("Pruebas en <HomeScree/>", () => {
  const user = {
    name: "Edgar",
    email: "edgar@mail.com"
  }
  // aqui el shallow solo renderiza el primer componente

  const wrapper = mount(
    <UserContext.Provider value={{user}}>
      <HomeScreen />
    </UserContext.Provider>
  );
  test("should displayed", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
