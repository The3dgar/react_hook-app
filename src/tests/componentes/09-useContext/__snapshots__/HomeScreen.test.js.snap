// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomeScree/> should displayed 1`] = `
<div>
  <h1>
    Home
  </h1>
  <hr />
  <pre>
    {"name":"Edgar","email":"edgar@mail.com"}
  </pre>
</div>
`;
